tests/helpers.py

```diff
@@ -88,14 +89,20 @@ def wait_for_resource_state(resource, state):
     return True
 
 
-def create_pod(interface_type=None, pvc_name=None, desired_status=constants.STATUS_RUNNING, wait=True):
+def create_pod(
+    interface_type=None, namespace=None, pvc_name=None, node_name=None,
+    desired_status=constants.STATUS_RUNNING, pod_dict=None, wait=True
+):
     """
     Create a pod
 
     Args:
         interface_type (str): The interface type (CephFS, RBD, etc.)
-        pvc (str): The PVC that should be attached to the newly created pod
+        namespace (str): The namespace in which pod to be created
+        pvc_name (str): The PVC that should be attached to the newly created pod
+        node_name (str): The name of specific node to schedule the pod
         desired_status (str): The status of the pod to wait for
+        pod_dict (str): Dictionary path for the pod
```

Comment:
confusing name:
pod_dict is str
Please change

Reply:
will change to pod_dict_path